aptos-move/mvhashmap/src/unit_tests/mod.rs

```diff
@@ -54,7 +54,8 @@ fn create_write_read_placeholder_struct() {
     let ap2 = KeyType(b"/foo/c".to_vec());
     let ap3 = KeyType(b"/foo/d".to_vec());
 
-    let mvtbl: MVHashMap<KeyType<Vec<u8>>, usize, TestValue, ExecutableTestType> = MVHashMap::new();
+    let mvtbl: MVHashMap<KeyType<Vec<u8>>, usize, usize, TestValue, ExecutableTestType> =
```

Comment:
Why is identifier type usize here instead of u64?

Reply:
No reason, it doesn't matter to much for unit tests, used usize like it was done for tags, can switch to u64 to make it clearer